dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -167,47 +166,11 @@ def refute_estimate(self):
         )
 
         return refute
-    def _save_precompute(self, pipeline):
-        pdb.set_trace()
-        precomputed_pipeline = []
-        isAtTheStart = True
-        new_outcome = self._data['y']
-        for action,props in pipeline:
-            if 'save_compute' in props and isAtTheStart:
-                precomputed_pipeline.append( ( action(new_outcome), {'input':['X'], 'save_compute':True} ) )
-                new_outcome = precomputed_pipeline[-1]( self._data[self._chosen_variables] )
-            else:
-                isAtTheStart = False
-                precomputed_pipeline.append( (action, props) )
-        
-        return precomputed_pipeline
-
-    def _parse_pipeline(self, pipeline):
-        function_pipeline = []
-        # Every element consists of the operation and the associated arguments
-        # (func, func_args)
-        for action, func_args in pipeline:
-            if callable(action): 
-                function_pipeline.append( (action, {'input':['X']}) )
-            elif action in DummyOutcomeRefuter.SUPPORTED_ESTIMATORS:
-                pdb.set_trace()
-                function_pipeline.append( ( lambda new_data: self._estimate_dummy_outcome(func_args, action, new_data), {'input':['X', 'y'], 'save_compute':False} ) )
-            elif action == 'noise':
-                function_pipeline.append( ( lambda new_data: self._noise(new_data, func_args), {'input':['y']} ) )
-            elif action == 'permute':
-                function_pipeline.append( ( lambda new_data: self._permute(new_data, func_args), {'input':['y']} ) )
-            elif action == 'zero':
-                function_pipeline.append( (lambda new_data: np.zeros(new_data.shape), {'input':['y']} ) )
-            else:
-                raise ValueError("Invalid action:{} passed to pipeline".format(action))
-
-        return function_pipeline
             
-    def _estimate_dummy_outcome(self, func_args, action, new_data):
-        pdb.set_trace()
+    def _estimate_dummy_outcome(self, func_args, action, new_outcome):

```

Comment:
the parameter can be simply called outcome, since it is the input. It is not new_outcome, especially if this function is the first action in the list.